test/reporter-stdout.js

```diff
@@ -0,0 +1,21 @@
+/*globals QUnit:true*/
+/*jshint node:true*/
+var QUnit = require( "../dist/qunit" );
+
+// Options: { output: "minimal" || "verbose" }
+QUnit.stdout();
+
+// Load QUnit tests
+require( "./logs" );
```

Comment:
It's certainly a not documented feature, and because of this it might be a bad practice. I'll insert an extra line exporting QUnit to the global scope.